api/src/main/java/com/valtech/aem/saas/api/query/SimpleFilter.java

```diff
@@ -15,15 +16,24 @@
 public class SimpleFilter implements Filter {
 
     private static final String FILTER_FIELD_VALUE_DELIMITER = ":";
+    public static final String QUOTE = "\"";
```

Comment:
you can inline the quote and it doesn't make sense to make this a public constant